src/Components/LoginSignUp/ParentLoginSignUp.jsx

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios'
+import React, { useContext, useEffect, useState } from 'react'
+import HeaderHomePage from '../Headers/HeaderHomePage'
+import LoginForm from './LoginForm'
+import SignUpForm from './SignUpForm'
+import { ToastContainer, toast } from 'react-toastify';
+import { useNavigate } from 'react-router-dom'
+import ContextAPI from '../../Context/ContextAPI'
+
+const ParentLoginSignUp = (props) => {
+    const context = useContext(ContextAPI)
+    console.log(context.usersData)
+    const navigate = useNavigate()
+    const [allUsers, setAllUsers] = useState([])
+    const [isLogggedIn, setIsLoggedIn] = useState([])
+    useEffect(() => {
+        const fetchData = async () => {
+            await axios.get('https://jsonplaceholder.typicode.com/users')
```

Comment:
axios methods should be placed in a separate file and imported into this file.

Reply:
Fixed